### PR TITLE
Allow namespace and projectname to be different in devepoment

### PR DESCRIPTION
**Pull request by Biboswan**
Once this is done I can update the library in notify and push that as well

### CODE REVIEW

**Comment by Biboswan** [comment]
Getting exception !!

File: src/publisher.js [hunk 1]
```diff
@@ -165,8 +168,10 @@ class PulsePublisher {
     this.exchanges = exchanges;
     this.sendDeadline = sendDeadline || 12000;
 
-    assert.equal(client.namespace, exchanges.projectName,
-      'client namespace must match projectName');
+    if (process.env.NODE_ENV ==='production') {
```
Comment (djmitche) [inline src/publisher.js]: space after =

File: src/publisher.js [hunk 2]
```diff
@@ -40,6 +39,10 @@ class Exchanges {
 
   async publisher({rootUrl, schemaset, client, sendDeadline, publish, aws}) {
     let publisher;
+    if (process.env.NODE_ENV ==='development') {
```
Comment (djmitche) [inline src/publisher.js]: Let's use `!== 'production'` here, to be the logical negative of the other conditoinal.  Also include the space :)

**Review by djmitche — COMMENTED** [review]
This looks good, just some minor fixes.

File: src/publisher.js (hunk 2 above)
Comment (Biboswan) [reply src/publisher.js]: why lint didn't catch that

**Comment by Biboswan** [comment]
ready

**Review by djmitche — APPROVED** [review]
Our lint configuration is pretty lame.  @owlishDeveloper is thinking about improving that, and could probably use some help..

**Comment by djmitche** [comment]
Will be released shortly as v2.0.3.

**Pull request merged**